src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;
+    }
+
+    @Override
+    public double getArea() {
+        return (firstSide * secondSide / 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(
+                "Figure: right triangle, area: " + getArea() + " sq. units"
+                        + ", first side: " + firstSide + " units"
+                        + ", second side: " + firstSide + " units"

```

Comment:
There is a typo in the `draw()` method. You are printing `firstSide` twice. The second one should be `secondSide`.